### PR TITLE
6.7 direct_geometry.rst release notes

### PR DESCRIPTION
**Pull request by jhaigh0**
**Description of work**

Edit the 6.7 direct geometry release notes making changes for grammar, spelling and relevance.

Note that empty sections are not removed until just before release to accommodate any last minute changes.

**To test:**

 - Build `docs-html`
 - Open the html file `/build/docs/release/v6.7.0/direct_geometry.html`
 - Check that all release notes make sense to you
 - Check that links go to the correct place
 - Be picky about stylistic consistencies

---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Comment by robertapplin** [comment]
Unassigned myself due to having problems building the docs-html target

File: docs/source/release/v6.7.0/direct_geometry.rst [hunk 1]
```diff
@@ -10,19 +10,19 @@ General
...
 
 Bugfixes
 ############
-- The peak picker tool is now offset on the y axis by the background amount.
-- The peak picker tool is now set as activate by default.
-- The settings loaded into the ALFView interface are now separate from those loaded into the InstrumentView.
-- Removing the sample on the ALFView interface will now clear the InstrumentWidget plot.
-- The 'Reset View' button on the 'Render' tab of ALFView will no longer prevent the 'Pick' tab tools from working.
+- The :ref:`ALFView <ALFView-ref>` peak picker tool is now offset on the y axis by the background amount.
+- The :ref:`ALFView <ALFView-ref>` peak picker tool is now set as activate by default.
```
Comment (MohamedAlmaki) [inline docs/source/release/v6.7.0/direct_geometry.rst:22]: Not sure, about this suggestion. Do you think it is right?
```suggestion
- The :ref:`ALFView <ALFView-ref>` peak picker tool is now activated by default.
```

File: docs/source/release/v6.7.0/direct_geometry.rst [hunk 2]
```diff
@@ -49,9 +49,9 @@ New features
 Bugfixes
 ############
 - Fixed GDOS intensity correction so that the correction is applied in the same way regardless of rotation of the slice plot.
-- Fix for error when saving a slice plot as a matlab file.
+- Fix for an error when saving a slice plot as a matlab file.
 - Warning now omitted if a cut is taken with a higher resolution than the parent slice. This causes the appearance of missing lines between adjacent datapoints.
-- When taking a cut, if a intensity range is provided it is now applied to the plot y limits.
+- When taking a cut, if an intensity range is provided it is now applied to the plot y limits.
 - When changing intensity of a slice plot the axis limits now get reset to their original values. This solves a bug where zooming out after an intensity change was not possible.
```
Comment (MohamedAlmaki) [inline docs/source/release/v6.7.0/direct_geometry.rst:55]:
```suggestion
- When changing the intensity of a slice plot the axis limits now get reset to their original values. This solves a bug where zooming out after an intensity change was not possible.
```

**Review by MohamedAlmaki — CHANGES_REQUESTED** [review]
Just a few typos

**Review by MohamedAlmaki — APPROVED** [review]
Looks, thanks for the changes!

**Pull request merged**